src/Education/Commands/ShowRuleHelpFromErrorList.cs

```diff
@@ -0,0 +1,91 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.ComponentModel.Composition;
+using System.ComponentModel.Design;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using SonarLint.VisualStudio.Core;
+using SonarLint.VisualStudio.Infrastructure.VS;
+
+namespace SonarLint.VisualStudio.Education.Commands
+{
+    public interface IShowHelpFromErrorList
+    { }
+
+    [Export(typeof(IShowHelpFromErrorList))]
+    [PartCreationPolicy(CreationPolicy.Shared)]
+    internal class ShowHelpFromErrorList : IShowHelpFromErrorList
+    {
+        private readonly IThreadHandling threadHandling;
+        private readonly IEducation education;
+        private readonly IErrorListHelper errorListHelper;
+
+        private readonly Guid showHelpCmdGuid = new Guid("4A9B7E50-AA16-11D0-A8C5-00A0C921A4D2");
```

Comment:
Probably worth adding a comment to make it clear that this is the MS-defined "Show Error Help" command.